pcm-player.js

```diff
@@ -85,6 +85,25 @@ PCMPlayer.prototype.createContext = function() {
     });
 };
 
+PCMPlayer.prototype.createContextWithElement = function() {
+    this.audioCtx = new (window.AudioContext || window.webkitAudioConext)();
```

Comment:
because a good portion of this logic is the same as `createContext`, perhaps we could apply it conditionally using `this.options.useAudioElement`, perhaps also moving the element-specific parts to a method like `PCMPlayer.prototype.createAudioElement`.  then we can remove the ternary logic in the constructor

Reply:
Yea, that's a good point. Will do.